spec/features/auth_users/auth_user_logs_out_spec.rb

```diff
@@ -0,0 +1,21 @@
+require "rails_helper"
+
+RSpec.feature "Authenticated user", type: :feature do
+  it "can logout" do
+    create(:user)
+
+    visit account_path
+    fill_in "session[email]", with: "richard@example.com"
+    fill_in "session[password]", with: "hello"
+    click_link_or_button "Login"
+    visit account_path
+    click_link_or_button "Sign Out"
+    within ("#flash_notice") do
+      expect(page).to have_content("Goodbye")
+    end
+    visit account_path
+    within ("#flash_alert") do
+      expect(page).to have_content("You must log in")
+    end
+  end
+end
```

Comment:
This feels weird being in it's own file, and the logic is a bit skewed. This is testing both login and logout, which means the user is unauthenticated at the beginning and end of the test. I think this would be better placed inside the `spec/features/user_login_spec.rb`